pandas/io/parsers.py

```diff
@@ -999,6 +999,8 @@ def _failover_to_python(self):
 
     def read(self, nrows=None):
         if nrows is not None:
+            nrows = _validate_integer('nrows', nrows)
```

Comment:
Where should I put this test? There are a bunch in `tests/io/parser', but nothing for read_csv directly.